src/gui/editors/AutomationEditor.cpp

```diff
@@ -1555,10 +1403,12 @@ float AutomationEditor::yCoordOfLevel(float level )
 
 
 
-				//NEEDS Change in CSS
+//NEEDS Change in CSS
```

Comment:
It looks like you've removed the part that needs a CSS change below, so remove this too?